models/eqFulfilment.go

```diff
@@ -0,0 +1,34 @@
+package models
+
+import "github.com/ONSdigital/census-rm-pubsub-adapter/validate"
+
+type EqFulfilment struct {
+	EqFulfilmentEvent   *RmEvent             `json:"event" validate:"required"`
+	EqFulfilmentPayload *EqFulfilmentPayload `json:"payload" validate:"required"`
+}
+
+type EqFulfilmentPayload struct {
+	FulfilmentRequest *FulfilmentRequest `json:"fulfilmentRequest" validate:"required"`
+}
+
+type FulfilmentRequest struct {
+	FulfilmentCode   string   `json:"fulfilmentCode" validate:"required"`
+	CaseID           string   `json:"caseId" validate:"required"`
+	IndividualCaseID string   `json:"individualCaseId"`
+	Contact          *Contact `json:"contact" validate:"required"`
```

Comment:
is it required or can it be null or not present?

Reply:
RM actually fails to process it without at least an empty `contact` block